src/yachtzee/buffs.ts

```diff
@@ -67,6 +68,13 @@ export function yachtzeePotionProfits(potion: Potion, yachtzeeTurns: number): nu
   return yachtzeeValue + embezzlerValue + barfValue - potion.price(true);
 }
 
+const doublingValue = (potion: Potion, yachtzeeTurns: number) =>
+  Math.min(
+    potion.price(false),
+    yachtzeePotionProfits(potion.doubleDuration(), yachtzeeTurns) -
+      Math.max(0, yachtzeePotionProfits(potion, yachtzeeTurns))
+  );
+
```

Comment:
```suggestion
const doublingValue = (yachtzeeTurns: number) => (potion: Potion) =>
  Math.min(
    potion.price(false),
    yachtzeePotionProfits(potion.doubleDuration(), yachtzeeTurns) -
      Math.max(0, yachtzeePotionProfits(potion, yachtzeeTurns))
  );
```

Would this look better as a curried function, letting us do `maxBy(doublingPotions, doublingValue(yachtzeeTurns))`?
This isn't actively a suggestion, just a pitch

Reply:
This is the same as `partial()` in Python, so it makes sense. I just don't know whether it's a common stylistic thing in JS/TS; if so I can change it